bin/download-syntax

```diff
@@ -212,9 +212,12 @@ REPOS = (
     ),
     Repo(
         name='microsoft/vscode',
-        version='b37252c1823',
+        version='df15c8e832a',
         license_path='LICENSE.txt',
-        grammars=('extensions/java/syntaxes/java.tmLanguage.json',),
+        grammars=(
+            'extensions/java/syntaxes/java.tmLanguage.json',
+            'extensions/swift/syntaxes/swift.tmLanguage.json',
```

Comment:
generally this should come from the source repository -- even the `java` one probably shouldn't be coming from there

usually you can find the source repository in a "comment" at the top of the microsoft one

Reply:
Problem with that is that i already tried the original source (`textmate/swift.tmbundle`) earlier, but  when i updating the grammars it replaces `HEAD` to ``. But i can try it.